src/components/Directory/utils.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
```

Comment:
i don't think we need this here because we're not using jsx here

Reply:
When I comment out this import I get this error
<img width="540" alt="Screen Shot 2021-04-23 at 1 53 16 PM" src="https://user-images.githubusercontent.com/25016486/115910843-56219800-a43b-11eb-9c25-639d11aba434.png">

Reply:
Resolved by changing location of SubheaderIcon : 17ee558